src/shared/components/mutationTable/column/VariantTypeColumnFormatter.tsx

```diff
@@ -0,0 +1,152 @@
+                className: 'variant-type-ins',
+            },
+            DEL: {
+                toolTip: 'Deletion',
+                className: 'variant-type-del',
+            },
+            CONSOLIDATED: {
+                className: 'variant-type-other',
+            },
+            OTHER: {
+                className: 'variant-type-other',
+            },
+        };
+    }
+
+    /**
+     * Gets the display value.
+     */
+    public static getDisplayValue(data: Mutation[]): string {
+        return VariantTypeColumnFormatter.getTextValue(data);
+    }
+
+    /**
+     * Gets the Text Value.
+     */
+    public static getTextValue(data: Mutation[]): string {
+        let textValue: string = '';
+        const dataValue = VariantTypeColumnFormatter.getData(data);
+
+        if (dataValue) {
+            textValue = dataValue.toString().toUpperCase();
+        }
+
+        return textValue;
+    }
+
+    /**
+     * Gets the actual data.
+     */
+    public static getData(data: Mutation[]) {
+        if (data.length > 0) {
+            return data[0].variantType;
+        } else {
+            return null;
+        }
+    }
+
+    /**
+     * Gets the Map Entry.
+     */
+    public static getMapEntry(data: Mutation[]) {
+        const variantType = VariantTypeColumnFormatter.getData(data);
+
+        if (variantType) {
+            return VariantTypeColumnFormatter.MAIN_VARIANT_TYPE_MAP[
+                variantType.toUpperCase()
+            ];
+        } else {
+            return undefined;
+        }
+    }
+
+    /**
+     * Gets the associated tooltip.
+     */
+    public static getTooltip(data: Mutation[]) {
+        const value:
+            | ICategoricalColumn
+            | undefined = VariantTypeColumnFormatter.getMapEntry(data);
+        if (value && value.toolTip) {
+            return value.toolTip;
+        } else {
+            return undefined;
+        }
+    }
+
+    /**
+     * Gets the CSS Class Name.
+     */
+    public static getClassName(data: Mutation[]): string {
+        const value:
+            | ICategoricalColumn
+            | undefined = VariantTypeColumnFormatter.getMapEntry(data);
+
+        if (value && value.className) {
+            return value.className;
+        }
+        // for unmapped values, use the "OTHER" style
+        else {
+            return VariantTypeColumnFormatter.MAIN_VARIANT_TYPE_MAP['OTHER']
+                .className;
+        }
+    }
+
+    public static renderFunction(data: Mutation[]) {
+        // use text for all purposes (display, sort, filter)
+        const text: string = VariantTypeColumnFormatter.getDisplayValue(data);
+        const className: string = VariantTypeColumnFormatter.getClassName(data);
+
+        // extract the tooltip, if available
+        const toolTip = VariantTypeColumnFormatter.getTooltip(data);
+        let content = <span className={styles[className]}>{text}</span>;
+
+        if (toolTip != undefined) {

```

Comment:
you can just do 
`if (toolTip)`
as an aside, we should always use !==  so that we never worry about type coercion.  double equals means no type coercion.    so, just always use === and !== in conditions

Reply:
thanks.  will do.

Reply:
Now fixed.